frontend/public/components/factory/table.tsx

```diff
@@ -270,6 +270,7 @@ const VirtualBody: React.FC<VirtualBodyProps> = (props) => {
     scrollTop,
     width,
     getRowProps,
+    namespace,
```

Comment:
We should find a way to implement this without needing to pass through a new `namespace` prop. if this needed if you write your own AppliedClusterResource table instead of using the default table?

Reply:
The actual ResourceQuota page doesn't use the DefaultList component like the Search and API Explorer pages. The changes in table.tsx were specifically for the actual ResourceQuota page. There wasn't a more straightforward way that I could see to get the namespace to the ResourceQuotaTableRow. It doesn't have access to the `match` URL information.

Reply:
I'm totally open to suggestions; I'm just not seeing anything we can do otherwise unless we're able to somehow feed the namespace to the ResourceQuotaTableRow through Redux or local storage.